src/test/java/com/collections/MyArrayListTest.java

```diff
@@ -0,0 +1,50 @@
+package com.collections;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class MyArrayListTest  {
+    private MyArrayList myArrayList;
+
+    @Before
+    public void setUp() throws Exception {
+        myArrayList = new MyArrayList<>();
+    }
+
+    @Test
+    public void getSize() {
+        myArrayList.add("One").add("Two").add("Three");
+        int expectedSize = 3;
+        assertEquals(expectedSize, myArrayList.getSize());
+    }
+
+    @Test
+    public void add() {
+        String value = "test";
+        MyArrayList expectedList = new MyArrayList();
+        expectedList.add(value);
+        myArrayList.add(value);
+        assertNotEquals(expectedList, myArrayList);
```

Comment:
why these arrays not equals, they consist the same value
maybe you need to use assertArrayEquals method
but it is also not good, because even method add() dont work, arrays will be equals
please think about how to correct logic of this test method